src/engine/strat_engine/backstore/devices.rs

```diff
@@ -228,6 +229,7 @@ pub fn initialize_devices(
     devices: Vec<DeviceInfo>,
     pool_uuid: PoolUuid,
     mda_data_size: MDADataSize,
+    key_description: Option<&str>,
```

Comment:
Is it possible to add a test that can make use of the ```key_description``` parameter _within_ this module?

Reply:
This depends on what you want to test. If we're testing anything like whether or not a device gets encrypted, we do already have a test for that in the `crypt` module and we will introduce issues around setting up the keyring. We can probably do that but it seems slightly redundant if we already do that elsewhere. If you'd like to test something like teardown, we may still have to set up the keyring for that but that seems less redundant to me. What are your thoughts on this?